ci/test-cross-image.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
```

Comment:
I've added this so we can test docker-in-docker scenarios where the `CROSS_CONTAINER_IN_CONTAINER` envvar has already been set.